### ISSUE TITLE
Wrong namespace for SchoolContext class

### ISSUE THREAD

**Issue by KirkMunro** [aspnet/AspNetCore.Docs#7745]
In part one of this tutorial, you can see that SchoolContext is in the ContosoUniversity.Models namespace. In this part of the tutorial, the first code file you need to create includes a using statement for ContosoUniversity.Data. That using statement should be for ContosoUniversity.Models instead, according to how the tutorial is written right now. I would have changed the file myself in a PR, but looking around at the sample cs files in GitHub, in there it seems that SchoolContext is in fact defined in the ContosoUniversity.Data namespace, so something's not set up right somewhere.

Also, general feedback on the two ASP.NET Core 2.1 Razor Pages tutorials: the first one is solid, but the second one needs quite a bit of rework, especially as you get into parts 6 and later.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 140ce637-4d58-995f-c69e-b1de51f1dbb5
* Version Independent ID: 9bc46ffa-60db-bdee-0984-fab183c8f503
* Content: [Razor Pages with EF Core in ASP.NET Core - Update Related Data - 7 of 8](https://docs.microsoft.com/en-us/aspnet/core/data/ef-rp/update-related-data?view=aspnetcore-2.1)
* Content Source: [aspnetcore/data/ef-rp/update-related-data.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/data/ef-rp/update-related-data.md)
* Product: **aspnet-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by guardrex** [comment]
The file for the code sample in this topic is here ...

https://github.com/aspnet/Docs/blob/master/aspnetcore/data/ef-rp/intro/samples/cu/Pages/Courses/DepartmentNamePageModel.cshtml.cs

... and the `SchoolContext` is in the `ContosoUniversity.Data` namespace here ...

https://github.com/aspnet/Docs/blob/master/aspnetcore/data/ef-rp/intro/samples/cu/Data/SchoolContext.cs

These files are associated with the "cu" version of the sample, a 2.0 sample ...

https://github.com/aspnet/Docs/tree/master/aspnetcore/data/ef-rp/intro/samples/cu

Earlier in the tutorial tho, there were upgrades to use the 2.1 version of the sample at ...

https://github.com/aspnet/Docs/tree/master/aspnetcore/data/ef-rp/intro/samples/cu21

That's the problem ... the tutorial draws its text/code from *two different samples doing things different ways*.

**Comment by bill1109** [comment]
Hey I've worked my way through Parts 1-6 and have had difficulty with understanding the proper namespace definitions for SchoolContext... what should it be and how should it have been dealt with in this currently published tutorial?  I've learned a lot re-doing the tutorials 1-6, but it's frustrating  to say the least.  I'm sure I'm not the only person to experience this, even if they chalk it up to themselves (which may be even worse).
